scripts/eslint/tests/argument-trivia.test.cjs

```diff
@@ -56,28 +62,25 @@ fn.apply(null, true);
 const fn = (prop: null) => {};
 fn(null);
             `,
-            errors: [{ messageId: "booleanTriviaArgumentError" }],
+            errors: [{ messageId: "argumentTriviaArgumentError" }],
         },
         {
             code: `
 const fn = (prop: boolean) => {};
 fn(false);
             `,
-            errors: [{ messageId: "booleanTriviaArgumentError" }],
+            errors: [{ messageId: "argumentTriviaArgumentError" }],
         },
         {
             code: `
 const fn = (prop: boolean) => {};
 fn(/* boolean arg */false);
             `,
-            errors: [{ messageId: "booleanTriviaArgumentSpaceError" }],
-        },
-        {
-            code: `
+            errors: [{ messageId: "argumentTriviaArgumentSpaceError" }],
+            output:`
 const fn = (prop: boolean) => {};
-fn(/* first comment */ /* second comment */ false);
-            `,
-            errors: [{ messageId: "booleanTriviaArgumentError" }],

```

Comment:
This case is now "okay" because the lint takes the _last_ comment as the hint, which I think is correct and is needed to handle cases where we put other comments before the parameter.